package.json

```diff
@@ -36,6 +36,9 @@
     ]
   },
   "dependencies": {
-    "@curveball/core": "0.11"
+    "@curveball/accesslog": "^0.2.0",
+    "@curveball/bodyparser": "^0.4.6",
+    "@curveball/core": "^0.11.1",
+    "@curveball/problem": "^0.2.1"
```

Comment:
@evert What do you think about having controller, router & typescript already installed in the starter package?

Reply:
I agree! I wanted to have a single endpoint (using a controller) that just emits some random stuff, so people know exactly how to continue from there.